best-practices/automl/README.md

```diff
@@ -0,0 +1,12 @@
+# Best Practices - AutoML
```

Comment:
```suggestion
# Best Practices - H2O AutoML
```

Reply:
Edit: To be clear from the beginning that we're specifically talking about H2O AutoML.